tensorflow/python/ops/image_ops_impl.py

```diff
@@ -475,6 +488,19 @@ def flip_up_down(image):
   Outputs the contents of `image` flipped along the height dimension.
 
   See also `reverse()`.
+    
+  Usage Example:
+
+  >>> x = [[[1.0, 2.0, 3.0],
+  ...       [4.0, 5.0, 6.0]],      
+  ...     [[7.0, 8.0, 9.0],      
```

Comment:
And here

Reply:
> I think it is because of the trailing whitespace confusing the automated import (code after formatting doesn't look the same as before). Can you fix this please? I marked a few places but there are more

All should be fixed thanks to [this](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) cool thing, all seems to be disappeared now.

Reply:
I always configure my editors to either strip whitespace on save or to show it using special characters.

Reply:
Yeah that makes sense, I will now configure it like that too, thanks for the tip!